gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/reactive/handlers/api/el/ContentTemplateVariableProvider.java

```diff
@@ -53,6 +54,8 @@ public class ContentTemplateVariableProvider implements ExecutionContextTemplate
     protected static final String TEMPLATE_ATTRIBUTE_RESPONSE_CONTENT_JSON = TEMPLATE_ATTRIBUTE_RESPONSE + ".jsonContent";
     protected static final String TEMPLATE_ATTRIBUTE_RESPONSE_CONTENT_XML = TEMPLATE_ATTRIBUTE_RESPONSE + ".xmlContent";
 
+
+    private static final Pattern EXTRACTING_ROOT_TAG_PATTERN = java.util.regex.Pattern.compile("(?s)(?:<\\?[^>]*>\\s*)?<([^?!/][^>]*)>.*");
```

Comment:
Just a suggestion: as performance purpose greedy quantifiers are more performant

```suggestion
    private static final Pattern EXTRACTING_ROOT_TAG_PATTERN = java.util.regex.Pattern.compile("(?s)(?:<\\?[^>]*+>\\s*+)?<([^?!/][^>]*+)>.*+");
```

I’m not sure that the end of regex is revelant (the part after the last >) because this force the regex engine to read all the content.